_global/css/board_front/growth-options.css

```diff
@@ -214,6 +214,10 @@ growth-cell icon.reclaim-one {
   margin-bottom: 7px;
 }
 
+growth-cell icon.reclaim-none {
+  margin-bottom: 7px;
+}
+
```

Comment:
This is a joke icon from my Water Acts Not spirit. We don't need to merge this particular change (not sure if you can accept changes line-by-line).